apps/docs/pages/guides/platform/oauth-apps/oauth-scopes.mdx

```diff
@@ -0,0 +1,43 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'oauth-scopes',
+  title: 'Scopes for your OAuth App',
+  description: 'Scopes let you specify the access your integration needs',
```

Comment:
```suggestion
  description: 'Scopes let you specify the access your integration needs',
```

Maybe "type of access" or "level of access" instead? "Access" is already granted, scopes only restrict to what extent.